interface/themes/colors/openemr5/login.scss

```diff
@@ -8,6 +8,9 @@
   border-top: solid $dark 40px;
   margin-top: 5%;
   max-width: 500px;
+  $black: #000 !default;
+  border-radius: 10px;
+  box-shadow: 0 30px 60px 0 rgba($black,0.3);
```

Comment:
$black shouldn't be defined here as that variable gets imported from `color_base.scss`

Reply:
SCSS has different files that create an object-oriented structure that eventually gets compiled into one CSS file. If you look around, `@import`'s show which files go into which files which go into one big file.

Reply:
That makes perfect sense. Here are the new changes.

Reply:
@juggernautsei The above code that I mentioned still hasn't been changed